code/datums/mood_events/generic_negative_events.dm

```diff
@@ -221,3 +221,6 @@
 	description = "<span class='boldwarning'>HE'S CUTTING ME OPEN!!</span>\n"
 	mood_change = -8
 
+/datum/mood_event/type_bait
+	description = "<span class='warning'>I caught that fish mid-conversation... I cant belive I did that...</span>\n"
```

Comment:
can't believe

Reply:
> can't believe

You know you can actually edit the line to let them just commit your suggestion

Reply:
no, i don't